tests/integration/test_dbapi_integration.py

```diff
@@ -311,7 +311,7 @@ def test_datetime_with_numeric_offset_time_zone_query_param(trino_connection):
     rows = cur.fetchall()
 
     assert rows[0][0] == params
-    assert cur.description[0][1] == "timestamp with time zone"
+    assert cur.description[0][1] == "timestamp(6) with time zone"
```

Comment:
Also add similar tests for `time` and `time with time zone`. Seems the description is not tested there.

Reply:
Actually there are already `time` and `time with time zone` tests, e.g. `test_time_query_param`, `test_time_with_named_time_zone_query_param`

Reply:
I meant there is no description test in those tests. Please add it to validate the type is correctly exposed.

Reply:
Done